test: add unit tests for pkg/common/utils

orthur2 opened the pull request:
#### What type of PR is this?
 
test


#### Check the PR title.
<!--
The description of the title will be attached in Release Notes, 
so please describe it from user-oriented, what this PR does / why we need it.
Please check your PR title with the below requirements:
-->
- [X] This PR title match the format: \<type\>(optional scope): \<description\>
- [X] The description of this PR title is user-oriented and clear enough for others to understand.



#### (Optional) Translate the PR title into Chinese.
测试：为pkg/common/utils添加测试



#### (Optional) Which issue(s) this PR fixes:
<!--
Automatically closes linked issue when PR is merged.
Eg: `Fixes #<issue number>`, or `Fixes (paste link of issue)`.
-->
#257

orthur2 committed:
Add unit tests for pkg/common/utils

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/cloudwego/hertz?pullRequest=1310) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/cloudwego/hertz?pullRequest=1310) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/cloudwego/hertz?pullRequest=1310) it.</sub>

orthur2 commented:
I don't seem to find information about how to check test coverage in CONTRIBUTING.md. Then, after submitting this PR, codedev didn't perform test coverage detection. Then I found the following error displayed at https://app.codecov.io/gh/cloudwego/hertz/pull/1310: 

Missing Head Commit
Unable to compare commits because the head commit was not found


I would really appreciate it if someone could tell me what I did wrong.

orthur2 committed:
test: add unit tests for pkg/common/utils #1310

What type of PR is this?
test

Check the PR title.
 This PR title match the format: <type>(optional scope): <description>
 The description of this PR title is user-oriented and clear enough for others to understand.
(Optional) Translate the PR title into Chinese.
测试：为pkg/common/utils添加测试

(Optional) Which issue(s) this PR fixes:

xiaost commented:
@orthur2 thanks for contribution. all changes submitted needs to be approved before running tests.
